src/config.rs

```diff
@@ -101,6 +104,9 @@ pub struct Address {
 
     /// Address stats
     pub stats: Arc<AddressStats>,
+
+    /// Number of errors encountered since last successful checkout
+    pub error_count: Arc<AtomicU64>,
```

Comment:
Can we show this in admin? This would be a really cool metric to expose.

Reply:
Shouldn't be too difficult to expose it. It gets reset with every successful checkout so I am not sure how useful it would be.

Reply:
Ah I see. Not so useful then, unless the replica is banned? Could be useful to see how many errors it took before it got successfully banned.

Reply:
I thought about not resetting it too but that meant errors that occurred 2 days ago can affect routing decisions today

Reply:
Good point.